phpBB/styles/prosilver/template/login_forum.html

```diff
@@ -22,7 +22,7 @@ <h2 class="login-title">{L_LOGIN}</h2>
 
 			<dl>
 				<dt><label for="password">{L_PASSWORD}{L_COLON}</label></dt>
-				<dd><input type="password" tabindex="1" id="password" name="password" size="25" class="inputbox narrow" autocomplete="off" /></dd>
+				<dd><input type="password" tabindex="1" id="password" name="password" size="25" class="inputbox narrow" autocomplete="current-password" /></dd>
```

Comment:
This is for forum passwords and not the user password

Reply:
ooops thanks i'll fix that